webpack.config.js

```diff
@@ -0,0 +1,39 @@
+var webpack = require('webpack');
+var BrowserSyncPlugin = require('browser-sync-webpack-plugin');
+var UglifyJsPlugin = webpack.optimize.UglifyJsPlugin;
+var env = process.env.WEBPACK_ENV;
+
+var plugins = [];
+
+if (env === 'build') {
+  // set NODE_ENV=production in environment,
+  // which ends up reducing size of React
```

Comment:
Are we using React somewhere?

Reply:
No, I think Alec copied this from the webpack config I used for Griduniverse, and I had based parts of that on another project that used React. Should be okay to remove this plugin.

Reply:
Indeed, this is just copied from the GU webpack config. For whatever reason Uglify doesn't seem to be compressing anything at this point (I suspect we may need to add some `babel` support to get uglify working), so this does seem pretty pointless. I don't think it's really React specific despite the comment though.

Reply:
Okay.